### PR TITLE
Use unittest.mock instead of case

### PR DESCRIPTION
**Pull request by matusvalo**

### CODE REVIEW

**Comment by matusvalo** [comment]
Hmm seems that tests are failing for python 3.5 :-/

**Comment by auvipy** [comment]
can you check the flake8

**Comment by matusvalo** [comment]
flake8 fixed...

File: t/integration/test_integration.py [hunk 1]
```diff
@@ -2,7 +2,10 @@
 from struct import pack
 
 import pytest
-from case import ANY, Mock, call, patch
+# python 3.5 does not support assert_called()
```
Comment (auvipy) [inline t/integration/test_integration.py:5]: master is 3.6+ only

**Review by auvipy — COMMENTED** [review]
3.6+only should be ok

**Pull request merged**